Cargo.toml

```
@@ -38,6 +38,8 @@ mimalloc = { version = "0.1.38", default-features = false }
 async-once-cell = {version="0.5.3", default-features=false}
 actix-governor = {version="0.5.0", default-features=false}
 mini-moka = { version="0.10", optional = true, default-features=false, features=["sync"]}
+brotli = { version = "3.4.0", default-features = false, features=["std"] }
+chacha20poly1305={version="0.10.1", default-features=false, features=["alloc","getrandom"]}
```

```suggestion
brotli = { version = "3.4.0", default-features = false, features=["std"], optional=true}
chacha20poly1305={version="0.10.1", default-features=false, features=["alloc","getrandom"], optional=true}
```

I would suggest adding the `optional=true` after both this crates because this will be required when adding the feature/conditional compiling feature approach in the project.

For a guide on how to provide conditional compilation approach in rust, I would suggest watching this video: 

https://piped.video/watch?v=QkHhwYJaP0U

---

Thank you for the feedback!